sdk/ml/azure-ai-ml/tests/test_configs/components/helloworld_inline_pipeline_component.yml

```diff
@@ -27,7 +27,7 @@ inputs:
 jobs:
   component_a_job:
     command: echo "hello" && echo "world" > ${{outputs.world_output}}/world.txt
-    environment: azureml:AzureML-sklearn-1.0-ubuntu20.04-py38-cpu@latest
+    environment: azureml:AzureML-sklearn-1.0-ubuntu20.04-py38-cpu:33
```

Comment:
this is a known issue tracked here: https://teams.microsoft.com/l/message/19:9149b12f63984624a9727423618746c8@thread.skype/1679552903070?tenantId=72f988bf-86f1-41af-91ab-2d7cd011db47&groupId=1a39822b-118a-450e-893b-32cdb76f1538&parentMessageId=1679552903070&teamName=STCA%20AI%20Platform%20Team&channelName=AML%20Pipelines&createdTime=1679552903070&allowXTenantAccess=false